apps/wallet-mobile/src/features/Send/useCases/ListAmountsToSend/ListAmountsToSendScreen.tsx

```diff
@@ -108,8 +103,10 @@ export const ListAmountsToSendScreen = () => {
     // NOTE: update on multi target support
     createUnsignedTx([toYoroiEntry(targets[selectedTargetIndex].entry)], {
       onSuccess: (yoroiUnsignedTx) => {
-        yoroiUnsignedTxChanged(yoroiUnsignedTx)
-        navigateToTxReview({unsignedTx: yoroiUnsignedTx, onSuccess, onError})
+        unsignedTxChanged(yoroiUnsignedTx)
+        onSuccessChanged(onSuccess)
+        onErrorChanged(onError)
```

Comment:
`onError` and `onSuccess` variables will be removed when product designs common screens